lib/ui/painting.dart

```diff
@@ -271,6 +271,7 @@ enum FilterQuality {
 
 /// Styles to use for line endings.
 ///
+/// Must be in the same order as the SkPaint::Cap enum.
```

Comment:
This shouldn't be a dartdoc, it won't make sense to our users. It should be a regular comment (and indeed there already is such a comment two lines later).